docs/google-test.md

```diff
@@ -0,0 +1,205 @@
+# Google Test Demo
+
+## Introduction
+
+[Google Test](https://github.com/google/googletest) is a unit testing framework for C++ programming developed by Google. It is currently being used for unit testing in the [Open Telemetry CPP](https://github.com/open-telemetry/opentelemetry-cpp) repo, along with tools like [codecov.io](http://codecov.io/) to thoroughly test the code in the repository. A primer for using Google Test can be found [here](https://github.com/google/googletest/blob/master/googletest/docs/primer.md).
```

Comment:
CPP -> C++

Reply:
Fixed

Reply:
Other way around please. :) 

Currently: Open Telemetry CPP
What I'd like: OpenTelemetry C++

Reply:
Ah, my mistake, I've made the appropriate changes now